src/index.js

```diff
@@ -0,0 +1,80 @@
+import './style.css';
```

Comment:
Hi, @Jones 
Great job so far. 
No linter errors used workflow in a professional way.
Clear code base and very professional.
Your code is a sample of DRY and not KISS.
Code is simple, readable, and not hardcoded at all.